sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/ParameterLoader.cs

```diff
@@ -107,7 +107,7 @@ private static ImmutableList<RuleParameterValues> ParseParameters(XDocument xml)
 
             var propertyParameterPairs = parameteredAnalyzer.GetType()
                 .GetTypeInfo()
-                .DeclaredProperties

```

Comment:
Was not taking properties of base classes.

Reply:
Is that the fix? Why do we need all the other changes then? ;)